### ISSUE TITLE
Re-enable Cinny subpath deployment

### ISSUE THREAD

**Issue by luixxiul** [spantaleev/matrix-docker-ansible-deploy#3719]
**Is your feature request related to a problem? Please describe.**

This is a tracking issue for https://github.com/spantaleev/matrix-docker-ansible-deploy/issues/3701.

https://github.com/spantaleev/matrix-docker-ansible-deploy/commit/446e6564247a87ac5c39bb3fb3ed4a0fb554de53 disabled Cinny subpath deployment due to the upstream's new requirement that you would need to rebuild the app by yourself after updating the base path. Per the comment on the commit the change looks to be temporary (hopefully) rather than permanent, waiting for future work.

**Describe the solution you'd like**

Re-enable Cinny subpath deployment.

**Describe alternatives you've considered**

**Additional context**

**Comment by spantaleev** [comment]
Closing this as part of a backlog cleanup.

Re-enabling subpath deployment remains blocked by upstream: Cinny's maintainers closed cinnyapp/cinny#1843 explaining that running under a subpath requires rebuilding the app with a changed base in its build config, and that prebuilt releases cannot support arbitrary subpaths. Forcing a self-build on everyone who wants a subpath is not practical either (the build is reported to be quite memory-hungry).

Also, the Cinny role has since been extracted to the mother-of-all-self-hosting organization (https://github.com/mother-of-all-self-hosting/ansible-role-cinny), which is where the path-prefix restriction now lives. Any future work to support subpath deployment would happen in that role, not in this playbook.

If upstream makes subpath deployment possible with prebuilt releases, or a contributor comes up with a workable approach, this can be revived (ideally as an issue against the MASH role).
